packages/nimble-components/src/table-column/mixins/fixed-width-column.ts

```diff
@@ -0,0 +1,38 @@
+import { attr, nullableNumberConverter } from '@microsoft/fast-element';
```

Comment:
@atmgrifter00 It'd be good for you to take a closer look at the fixed width column and its tests. I just skimmed it and saw no problems, but you know this area far better than I do.